docs/events.md

```diff
@@ -0,0 +1,175 @@
+# Events list
```

Comment:
be careful on events names

ex: leaving, leave, left, move , moving, moved has very different meanings, you should adopt a convention
request for an action : 'move'
acknowledge | inform about an action: 'moved'